public/index.html

```diff
@@ -4,11 +4,23 @@
   <link rel="stylesheet" type="text/css" href="styles/interface.css">
   <script src="/socket.io/socket.io.js"></script>
   <script src="scripts/main.js"></script>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
 </head>
 
 <body>
-  <input id="input" autocomplete="off" /><button onClick="sendMessage()">SEND</button>
+
+  
   <ul id='messages'></ul>
 </body>
+<footer>
+  <input class="input" id="input" autocomplete="off" action="sendMessage()" /><button class="button" id="button" onClick="sendMessage()">SEND</button>
```

Comment:
I thought you needed 'class' for css. Guess not! Removed.

Lol, forgot I put that in there. Does nothing. Removed.

Not sure what the best practice is, but good to be consistent. I'm fine with that. I moved it.